scripts/build_snap.sh

```diff
@@ -7,7 +7,14 @@ if [ ! -d "dvc" ]; then
   exit 1
 fi
 
-sudo snapcraft --use-lxd
+sg lxd -c snapcraft
+original_snap_name="$(ls dvc_*.snap)"
+mv "$original_snap_name" original.snap
+
+git apply scripts/remove_update_warning.patch
```

Comment:
This will make `dvc --version` show up as dirty

Reply:
yes; I don't think that's an issue (`dvc-0.95.0-1+gs0m3sha`).